### DISCUSSION TITLE
Dane o wynikach kampanii Ads Allegro, wznawianie i wyłączanie kampanii Ads Allegro

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#8645] [Suggestion - verification]
Dostęp po publicznym API Allegro do Ads Allegro:
 - odczyt wyników kampanii Ads Allegro (kliknięcia, koszty, itp.),
 - zgłaszane już wcześniej - możliwość tworzenia, wyłączenia i wznowienia kampanii Ads Allegro.

**Comment by cgchow94** [comment]
is there any updates here?

**Reply by Lukasz-Zurek** [reply]
No decisions have been made here yet.

**Reply by tomasz-grzesik** [reply]
Przydałoby się chociażby do ustawienia kampanii wg harmonogramu.

**Reply by Lukasz-Zurek** [reply]
Przypomnimy się noworocznie z tym tematem. Dziękuję.
